views/pages/about.ejs

```diff
@@ -38,33 +63,44 @@
       <img class="w-2/3" src="/images/about-hero.png" alt="no image" />
     </div>
     <div class="text-lg py-4 text-center">
-      <b>Welcome to Mind Drive</b>, <br />Your dedicated companion on your
-      journey towards mental well-being.
+      <b>Welcome to Mind Drive</b>, <br />Your dedicated companion on your journey towards mental well-being.
     </div>
     <div class="">
       <span class="text-lg flex items-center font-bold justify-center py-2">What do you get here?</span>
       <ul class="list-disc pl-4 py-4">
         <li>Explore personalized pathways, engage in mindfulness practices.</li>
-        <li>
-          Track your progress, and discover expert insights tailored to your
-          needs.
-        </li>
+        <li>Track your progress, and discover expert insights tailored to your needs.</li>
       </ul>
 
       <div class="flex items-center justify-center">
-        Whether you're seeking calm, clarity, or connection, Mind Drive is here
-        to guide you every step of the way. Together, let's drive towards a
-        healthier mind and a brighter tomorrow.
+        Whether you're seeking calm, clarity, or connection, Mind Drive is here to guide you every step of the way. Together, let's drive towards a healthier mind and a brighter tomorrow.
       </div>
     </div>
 
-    
     <div class="text-sm items-center justify-center flex flex-col my-8">
-      Copyright © 2024. All rights reserved. 
+      Copyright © 2024. All rights reserved.
       <br>Sristy Paul & Pooranjoy Bhattacharya
     </div>
   </div>
+
   <%- include('../includes/navbar') %>
+
+  <script>
```

Comment:
Donot write scripts in the `views` files. We are maintaining a proper directory structure. Please write the scripts in `public/js` directory and link them in `views`